frontend/packages/console-dynamic-plugin-sdk/src/utils/flags.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import { action } from 'typesafe-actions';
+import { RootState } from '@console/internal/redux';
+
+export enum ActionType {
```

Comment:
Maybe we should move all non-hook stuff to [src/app/features](https://github.com/openshift/console/tree/master/frontend/packages/console-dynamic-plugin-sdk/src/app) ? We already have other parts of store there